ghost/link-replacement/lib/PostLink.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @typedef {import('bson-objectid').default} ObjectID
+ */
+
+class PostLink {
+    /** @type {ObjectID} */
+    post_id;
+    /** @type {ObjectID} */
+    link_id;
```

Comment:
@SimonBackx I went for just an ID for now just to get things moving, not sure if we'll eventually want to switch to a full `LinkRedirect` object

Reply:
Didn't we want to put the PostLink class in the link-tracking package?